<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contoller</class>
 <widget class="QWidget" name="Contoller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_ctrlMAX1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 251, 252);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 20);
border-top-left-radius:6px;
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;
</string>
   </property>
   <property name="text">
    <string>表压</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ctrlMAX2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 251, 252);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 20);
border-top-left-radius:6px;
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;
</string>
   </property>
   <property name="text">
    <string>绝压</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ctrlMAX3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>210</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 251, 252);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 20);
border-top-left-radius:6px;
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;
</string>
   </property>
   <property name="text">
    <string>表压/绝压</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>设备类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>稳定值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ctrlstab2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ctrlstab1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>% F.S.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>240</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>变化率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>慢速</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中速</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>快速</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Variable</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_ctrlvariable">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Low Overshoot</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>520</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reference Vacuum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ctrlMAX2_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>660</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>High Speed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ctrlMAX1_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>710</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 251, 252);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 20);
border-top-left-radius:6px;
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;
</string>
   </property>
   <property name="text">
    <string>重启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ctrlMAX3_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>660</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ctrlMAX3_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>660</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tune</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>-5</y>
     <width>1280</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 252, 254);
background-color: rgba(05, 02, 04,50);
</string>
   </property>
   <property name="text">
    <string>       菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>15</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/icon_菜单.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_configureMachine_2">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>10</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:6px;
background-color: rgba(99, 115, 187,75);
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>返  回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/1.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>checkBox_2</zorder>
  <zorder>checkBox</zorder>
  <zorder>btn_ctrlMAX2_2</zorder>
  <zorder>btn_ctrlMAX3_2</zorder>
  <zorder>btn_ctrlvariable</zorder>
  <zorder>btn_ctrlMAX3_3</zorder>
  <zorder>comboBox</zorder>
  <zorder>btn_ctrlstab2</zorder>
  <zorder>btn_ctrlstab1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>frame</zorder>
  <zorder>btn_ctrlMAX1</zorder>
  <zorder>btn_ctrlMAX2</zorder>
  <zorder>btn_ctrlMAX3</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>label_11</zorder>
  <zorder>btn_configureMachine_2</zorder>
  <zorder>btn_ctrlMAX1_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
